keepassxc-browser/content/fill.js

```diff
@@ -358,6 +358,9 @@ const showErrorNotification = async function(errorMessage, notificationType = 'e
     const connectedDatabase = await sendMessage('get_connected_database');
     if (!connectedDatabase?.identifier) {
         kpxcUI.createNotification('error', tr('errorCurrentDatabaseNotConnected'));
+    } else if (!await isIframeAllowed()) {
+        // Special error if we are blocking due to iframe
+        kpxcUI.createNotification('error', tr('credentialsBlockedInIframe'));
```

Comment:
This should probably be inside the `isIframeAllowed()` after the `logDebug()` for sandboxed iframe.

Reply:
isIframeAllowed is called all over the place and is done prior to credential retrieval from KeePassXC. It would not be appropriate to show a banner at this stage of the process. The banner notification should only show when you try to fill credentials.

Reply:
Sounds reasonable.